docs/learn/learn-staking.md

```diff
@@ -6,110 +6,323 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
-as nominators. Nominators may back up to 
+Here you will lean about what is staking, why is important and how it works for
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}.
+
+## Introduction
+
+Blockchain networks use [consensus](learn-consensus.md/#why-do-we-need-consensus) mechanisms to add
+blocks on the chain. Consensus is the process of agreeing on something, in this case the progression
+of the blockchain or how blocks are added to the chain. Consensus is split into two protocols:
+`block production`, i.e. the way multiple blocks candidates are produced, and `block finality`, i.e.
+the way only one block out of many candidates is selected and added to the chain. Because in public
+blockchains we have many participants who do not know each other (and probably never will), the
+reach of consensus happens in a secure and trustless way using Proof-of-Work (PoW) or Proof-of-Stake
+(PoS). In PoS networks like {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} the
+security of the network is dependent on the amount of capital locked on chain: the more the capital
+locked the lower the chance that someone will be able to successfully attack the network, as they
+would require a lot of own tokens or to collude different network participants. The process of
+locking tokens on the chain is also called `staking`. This is the main difference between PoS and
+PoW networks like Bitcoin that base their security on solving mathematic puzzles, a solution that
+has been criticized due to the high amount of energy needed for computers to solve such puzzles. In
+PoW networks miners are responsible for adding blocks to the chain, and for doing such work they are
+rewarded with tokens. Similarly to the miners in PoW networks, in PoS networks we have `validators`.
+Token holders can lock funds on chain and for doing so they are getting `staking rewards`. There is
+thus an economic incentive for token holders to become active participants who contribute to the
+security and economic stability of the network. PoS networks in general are therefore more inclusive
+than PoW networks, as participants do not need to have either the technical knowledge about
+blockchain technology nor the experience in running mining equipment. PoS ensures that everybody has
+"skin in the game" and thus can be held accountable. In case of misbehavior participants to the
+staking process can be punished or `slashed`, and depending on the gravity of the situation their
+stake can be partly or fully confiscated.
+
+## Nominated Proof-of-stake overview
+
+The two main types of PoS are Delegated PoS (DPoS) and Nominated PoS (NPoS).
+
+DPoS networks weigh validators by stake, oftentimes allowing the highest-stake validators to take
+disproportionate control of the network’s consensus protocol. In DPoS networks, delegators are not
+subject to loss of stake based on the behavior of the validator.
```

Comment:
Probably better to avoid mention of DPoS right at the beginning. Instead, can highlight what NPoS is and how it differs from other PoS mechanisms like DPoS later on in the doc https://www.gemini.com/cryptopedia/proof-of-stake-delegated-proof-of-stake-consensus-mechanism

I'm leaning towards retaining existing Wiki introduction to NPoS

```suggestion
{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} implements NPoS (Nominated Proof-of-Stake),
a relatively novel and sophisticated mechanism to select the validators who are allowed to participate in 
its [consensus](learn-consensus.md) protocol. The NPoS encourages {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} 
holders to participate as nominators.
```